argusclient/model.py

```diff
@@ -229,6 +229,34 @@ class Dashboard(BaseEncodable):
     def __init__(self, name, content, **kwargs):
         super(Dashboard, self).__init__(name=name, content=content, **kwargs)
 
+class Permission(BaseEncodable):
+    """
+    Represents a Permission object in Argus.
+
+    **Required parameters to the constructor:**
+
+    :param type: the type of permission - "user" or "group"
+    :type type: str
+    :param permissionNames: permissions (like "VIEW", "EDIT", "DELETE")
```

Comment:
Is it a bit unclear to me what this field is meant to be. Would this be considered metadata of what type of permissions that are supported? Could you point me to any existing documentation? Perhaps we could add more details here?

Reply:
added lots of details to the docs. check it out - that should answer your questions